app/assets/stylesheets/styles.scss

```diff
@@ -5,3 +38,99 @@
 .centered {
   text-align: center;
 }
+
+.goals-list {
+}
+
+.wordwrap {
+   white-space: pre-wrap;
+   white-space: -moz-pre-wrap;
+   white-space: -pre-wrap;
+   white-space: -o-pre-wrap;
+   word-wrap: break-word;
+}
+
+.goals-header {
+  text-align: center;
+  padding-bottom: 20px;
+}
+
+.add-goal {
+  padding-top: 50px;
+  padding-bottom: 20px;
+}
+
+.goal-details {
+  background-color: #ffffff;
+  font-family: Roboto, sans-serif;
+  box-shadow: 0 1px 6px rgba(0, 0, 0, 0.12), 0 1px 4px rgba(0, 0, 0, 0.24);
+  border-radius: 2px;
+  margin: 10px;
+  height: 50px;
+
+  a {
+    color: black;
+
+    .card {
+      background-color: inherit;
+    }
+
+    &:hover {
+      color: lighten(black, 40%);;
+    }
+  }
+
+  ul li {
+    list-style: none;
+    font-size: 16px;
+    padding-top: 13px;
+
+    .inline {
+      display: inline;
+    }
+  }
+
+  .goals-logo-red {
+    height: 40px;
```

Comment:
Properties should be ordered background-color, border-radius, color, display, font-size, height, text-align, width